src/tests/backend/specs/socketio.js

```diff
@@ -54,7 +54,7 @@ const connect = async (res) => {
   logger.debug('socket.io connecting...');
   const socket = io(`${common.baseUrl}/`, {
     forceNew: true, // Different tests will have different query parameters.
-    path: '/socket.io',
+    // path: '/socket.io',
```

Comment:
don't comment out, just remove.

We do use /socket.io path for query string but we can modify that no problem.